### ISSUE TITLE
Misconfiguration in curl query strings

### ISSUE THREAD

**Issue by kutysam** [vchinnipilli/kubestriker#31]
Hi,
Firstly, awesome project!

1. I can't seem to find FE and BE endpoint codes. Are they open sourced too? I am currently pulling from the docker container.
2. As you can see the image, there is no token`=` appended. Thus, token is not working. I've manually sent a curl with `=` and its working.
![image](https://user-images.githubusercontent.com/1703663/132180323-784c57aa-b28c-47c4-91c8-b7ee80ad5054.png)

Basically this happens when you are adding a Generic k8s cluster from FE ^. Should be an easy fix.
